### DISCUSSION TITLE
[tabulator-tables] Published versions with incorrect version number: v5.32.x

### DISCUSSION THREAD

**Discussion by jonatanschroeder** [DefinitelyTyped/DefinitelyTyped#66059] [Issues with a @types package]
The [npm repository for @types/tabulator-tables](https://www.npmjs.com/package/@types/tabulator-tables?activeTab=versions) includes versions 5.32.0 and 5.32.1 as older versions, when the new version is 5.4.8. This means that if I try to `yarn add -D @types/tabulator-tables` it will pick up 5.32.1 as the latest version, since it's numerically higher than 5.4.8. Can you please unpublish the older versions with incorrect numbers?

[a comment by an automated account is not shown]

**Reply by Jojoshua** [reply]
I believe the TS team needs to address that question. It autopublishes the version I believe.

**Comment by mike-lischke** [comment]
Side note: in VS Code I get the latest 4 minor version as suggestion, not 5.32:

<img width="707" alt="Bildschirmfoto 2023-07-14 um 08 51 16" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/2398704/15e3becf-08bf-4d38-aa10-d3107d3d9042">

**Reply by nwalters512** [reply]
Maybe this is only an issue with Yarn? `npm install @types/tabulator-tables` adds `^5.4.2` to `package.json` and installs version `5.4.2`. Yarn adds the same version range but installs `5.32.1`. Seems like they treat the `latest` tag differently.

**Reply by jonatanschroeder** [reply]
I believe yarn uses semver for comparison and ignores the latest tag unless explicitly asked for. `yarn add @types/tabulator-tables@latest` installs 5.4.8.
